data/mods/Tolerate_This/modinfo.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "type": "MOD_INFO",
+    "ident": "Tolerate_This",
+    "name": "Bens GF recipies",
```

Comment:
I suggest changing name of the mod in the future to make it more clear what it does from the first sight.

Reply:
Will do. I took too much influence from existing mods and names. Now it's more a serious attempt at some coding.

Though not sure if I'll be able to add anything else in the future. Seems someone may be working on some material flag/tag inheritance for ingredients (at least in foods). Which is great! :)